style.css

```diff
@@ -0,0 +1,148 @@
+body {
+  position: relative;
+  width: 375px;
+  height: 6715px;
+}
+
+.header {
+  display: flex;
+  position: absolute;
+  width: 100%;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.nav a {
+  text-decoration: none;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #27342e;
+  padding: 10px 12px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  position: absolute;
+  width: 375px;
+  height: 726px;
+  left: 0;
+  top: 92px;
+  background-image: url("assets/images/header-illsutration-mobile.png");
+}
```

Comment:
[OPTIONAL] Kindly adjust the background image to cover the whole with of the screen and add the propert `backgorund-repeat: no-repeat` which makes the background not repeated like the Figma design and adjust it's size.

Reply:
Thank you